site/topics/objects/introduction.rst

```diff
@@ -115,13 +120,13 @@ Constructor/Instantiation and Attributes
 
 * In the above example, we see the use of the special function called ``__init__``, which describes how to initialize an instance of the class
 
-    * The ``__init__`` method is called the *instantiation* method, but is commonly referred to as the *constructor*
+    * The ``__init__`` method is called the *constructor*, or the *instantiation* method
```

Comment:
Again, it's not _instantiation_. That doesn't even have _init_ in it.

Reply:
I always assumed it was initialization, but thought I saw the other today. Upon further investigation, no, it's just straight up initialization.

Reply:
I even thought to myself _init is a dumb choice for instantiation_